### DISCUSSION TITLE
When do weekly budgets update?

### DISCUSSION THREAD

**Discussion by cinderblockgames** [firefly-iii/firefly-iii#7558] [Firefly III] [answered]
I've got the cron running, and I can see that it's being called.  I expected my weekly rollover budget to update overnight, but it didn't.

**Comment by JC5** [comment, marked as answer]
Should be on mondays

**Reply by cinderblockgames** [reply]
Thank you!
Is there any way to change that to Sundays?

**Reply by JC5** [reply]
No, sorry.
